taxi/forms.py

```diff
@@ -39,14 +39,56 @@ def clean_license_number(self):
         return validate_license_number(self.cleaned_data["license_number"])
 
 
-def validate_license_number(
-    license_number,
-):  # regex validation is also possible here
-    if len(license_number) != 8:
-        raise ValidationError("License number should consist of 8 characters")
-    elif not license_number[:3].isupper() or not license_number[:3].isalpha():
-        raise ValidationError("First 3 characters should be uppercase letters")
-    elif not license_number[3:].isdigit():
-        raise ValidationError("Last 5 characters should be digits")
+def validate_license_number(license_number):
+    uppercase_char_num = 3
+    digit_char_num = 5
+    license_number_length = uppercase_char_num + digit_char_num
+
+    if len(license_number) != license_number_length:
+        raise ValidationError(
+            f"License number must consist of "
+            f"{license_number_length} characters!"
+        )
+
+    if not (
+            license_number[: uppercase_char_num].isalpha()
+            and license_number[: uppercase_char_num].isupper()
```

Comment:
remove the whitespace`[:uppercase_char_num]`
as we usually do something like this: `[1:3]`, not `[1: 3]`